client/layout/logged-out.jsx

```diff
@@ -2,36 +2,31 @@
  * External dependencies
  */
 var React = require( 'react' ),
-	classNames = require( 'classnames' );
+	classNames = require( 'classnames' ),
+	connect = require( 'react-redux' ).connect;
 
 /**
  * Internal dependencies
  */
 var Masterbar = require( './masterbar' ),
-	NoticesList = require( 'notices/notices-list' ),
-	notices = require( 'notices' );
+	GlobalNotices = require( 'components/global-notices' ),
+	notices = require( 'notices' ),
+	LoggedOutLayout;
 
-module.exports = React.createClass( {
+LoggedOutLayout = React.createClass( {
 	displayName: 'LayoutLoggedOut',
 
-	getInitialState: function() {
-		return {
-			section: undefined,
-			noSidebar: false
-		};
-	},
-
 	render: function() {
-		var sectionClass = this.state.section ? ' is-section-' + this.state.section : '',
+		var sectionClass = this.props.section ? ' is-section-' + this.props.section : '',
 			classes = classNames( 'wp', sectionClass, {
-				'has-no-sidebar': this.state.noSidebar
+				'has-no-sidebar': !this.props.hasSidebar
```

Comment:
Nit-picking: Should be a space after the `!` ([this'll eventually be part of a potential future custom ESLint rule](https://github.com/Automattic/wp-calypso/blob/8171f7a6002489800fa68be0ba5c88af5adde634/.eslintrc#L81-L82)).